src/test/java/ca/team2706/frc/robot/commands/drivebase/DriverAssistVisionTest.java

```diff
@@ -0,0 +1,130 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import ca.team2706.frc.robot.config.Config;
+import ca.team2706.frc.robot.subsystems.DriveBase;
+
+import jaci.pathfinder.Pathfinder;
+import jaci.pathfinder.Trajectory;
+
+import mockit.Expectations;
+import mockit.Mocked;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DriverAssistVisionTest {
+
+    private final DriverAssistVision driverAssistVision = new DriverAssistVision();
```

Comment:
This should be declared as 
```Java
@Tested
private DriverAssistVision driverAssistVision;
```
That way, it gets reinitialized before each test.

Reply:
Changed.